Tests/SwiftLintFrameworkTests/MultilineParametersRuleTests.swift

```diff
@@ -0,0 +1,52 @@
+import SwiftLintFramework
+import XCTest
+
+final class MultilineParametersRuleTests: XCTestCase {
```

Comment:
Instead of having this custom test case you can actually now have examples that define their configuration https://github.com/realm/SwiftLint/blob/f66c51b9f277b8b73c814f851594c1b3b010f2e2/Source/SwiftLintFramework/Rules/Lint/TestCaseAccessibilityRuleExamples.swift#L54

Reply:
Awesome! I'll update the PR later today.

Reply:
Done :-)